src/cmake/GeosxOptions.cmake

```diff
@@ -150,6 +148,10 @@ if( ${CMAKE_MAKE_PROGRAM} STREQUAL "ninja" OR ${CMAKE_MAKE_PROGRAM} MATCHES ".*/
   set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GEOSX_NINJA_FLAGS}" )
 endif()
 
+if( ENABLE_COV )
```

Comment:
I'm seeing `ENABLE_COV` and `ENABLE_COVERAGE` in different cmake files, are they different or intended to be the same/synonymous?

Reply:
not intended